fn_test.go

```diff
@@ -71,19 +71,32 @@ func TestRunFunction(t *testing.T) {
 				},
 			},
 		},
+<<<<<<< Updated upstream
 		"DatabaseInstance": {
 			reason: "The Function should return a fatal result if no input was specified",
 			args: args{
 				req: &fnv1beta1.RunFunctionRequest{
 					Meta: &fnv1beta1.RequestMeta{Tag: "database-instance"},
+=======
```

Comment:
Could you please resolve this conflict code?

Reply:
@Peefy should be good now :)